pricing.js

```diff
@@ -55,20 +55,40 @@ function calculateLTDPrice(product, employee, selectedOptions) {
   return price
 }
 
+function calculateCommuterCost(product, selectedOptions) {
+  let price = 0
+
+  if (selectedOptions.benefits === 'parking') {
+
+    return price += 250
+
+  }
+  else {
+    price += 84.75
+
+    return price
+  }
+}
 
-function calculateProductPrice(product, employee, selectedOptions) {
+function calculateProductPrice(product, selectedOptions, employee) {
```

Comment:
Interesting change, means you could have left employee of for the volLife integration tests since you don't use it there either.